plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -52,43 +54,27 @@ const columns: TableColumn<EntityRow>[] = [
     field: 'metadata.name',
     highlight: true,
     render: entity => (
-      <Link
-        component={RouterLink}
-        to={generatePath(entityRoute.path, {
-          ...entityRouteParams(entity),
-          selectedTabId: 'overview',
-        })}
-      >
-        {entity.metadata.name}
-      </Link>
+      <EntityRefLink entityRef={entity}>{entity.metadata.name}</EntityRefLink>
     ),
   },
   {
     title: 'System',
     field: 'row.partOfSystemRelationTitle',
     render: entity => (
-      <>
-        {entity.row.partOfSystemRelation && (
-          <EntityRefLink
-            entityRef={entity.row.partOfSystemRelation}
-            defaultKind="system"
-          />
-        )}
-      </>
+      <EntityRefLinks
+        entityRefs={entity.row.partOfSystemRelations}
+        defaultKind="system"
+      />
     ),
   },
   {
     title: 'Owner',
     field: 'row.ownedByRelationsTitle',
     render: entity => (
-      <>
-        {entity.row.ownedByRelations.map((t, i) => (
-          <React.Fragment key={i}>
-            {i > 0 && ', '}
-            <EntityRefLink entityRef={t} defaultKind="group" />
-          </React.Fragment>
-        ))}
-      </>
+      <EntityRefLinks
+        entityRefs={entity.row.ownedByRelations}
+        defaultKind="group"
```

Comment:
Can an owner be a kind `User` also?  The field looks to be optional, and by way of the name `defaultKind` suggests as so, but defining it explicitly here apols can't quite tell through the code flow.

https://backstage.io/docs/features/software-catalog/well-known-relations#ownedby-and-ownerof

If it harcodes group for now, that could be fine, may just be something to add a incremental update story about to add User later?  Maybe adding some elements to the tests to set different owner names.

Reply:
[Owner can bei either group or user](https://backstage.io/docs/features/software-catalog/descriptor-format#specowner-required) as defined in the spec. But it also defines the default kind.

But the `defaultKind` parameter just hides the kind in the link display text if the `entityRef` is the same as the default. The spec defines group to be the default kind, that means:

For a group called test it displays `test`
For a user called test it displays `user:test`

It's good that it is defined by the spec and not the relations alone because it depends on the actual usage of the relation. For example the `partOf` case is a bit more complicated: The parent can be a `system` or a `domain` depending on the actual case.

Reply:
thanks for the helpful description!!